### DISCUSSION TITLE
Clarification on traits

### DISCUSSION THREAD

**Discussion by Tequilaman** [Truinto/DarkCodex#93] [General]
Hi, in description of metamagic master it says you can reduce by 2 the final adjusted level, while in tabletop it would be 1. Also you'd be able to select up to level 3 spells (same for Magical Lineage). 

Have these traits been tweaked for the mod? I thought these would be homebrew but I disabled homebrew in the options, so now I'm not sure. 

Thanks

**Comment by Truinto** [comment]
Owlcat made it so you pick only on trait instead of two. But the traits are twice as useful. I kept that logic. Extra traits would grant 2 traits, you get 1 but it's like metamagic master and magical lineage together.

I was too lazy to add the level restriction. Picking a 3rd level spell seems like the optimal choice anyway.
